Gemfile.lock

```diff
@@ -172,7 +172,8 @@ GEM
     multi_test (0.1.2)
     multi_xml (0.5.5)
     multipart-post (2.0.0)
-    mysql2 (0.3.10)
+    mysql2 (0.3.20)
+    mysql2 (0.3.20-x86-mingw32)
```

Comment:
I just run the command to update the gem, It is not the only thing with two entries.

Does it need two, I expect not but I don't want to edit the Gemfile.lock file by hand as that seems wrong.